tests/testthat/test-parse.r

```diff
@@ -0,0 +1,52 @@
+context("Parsing expressions")
+
+expect_equal(
```

Comment:
These need to live a couple of `test_that()` blocks in `test-utilities.R` - see detail in http://style.tidyverse.org/tests.html#organisation (ggplot2 doesn't fully follow this organisation but should move towards it)

Reply:
Got it! I'll move the tests into `test-utilities.R`